### PR TITLE
docs: Create Reporting Guidelines markdown file

### LINKED ISSUE #14: docs: Create Reporting Guidelines markdown file
## Description
As a developer,
I need to see Reporting Guidelines for the repository
## Acceptance Criteria
File path  `./github/REPORTING_GUIDELINES.md`
## Update [Required]

- [ ] Create REPORTING_GUIDELINES.md - Reference: https://github.com/anitab-org/mentorship-android/blob/develop/.github/REPORTING_GUIDELINES.md
## Definition of Done

- [ ] All of the required items are completed.
- [ ] Approval by 2

## Estimation
1 hour

### PR DESCRIPTION
**Pull request by prachi237**
added reporting guidelines

### Description

Fixes #14

### Type of Change:

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged
- [ ] I have written Kotlin Docs whenever is applicable

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: .github/REPORTING_GUIDLINES.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+We'll respond within one week to the person who filed the report with either a resolution or an explanation of why the situation is not yet resolved.
+
+Once we've determined our final action, we'll contact the original reporter to let them know what action (if any) we'll be taking. We'll take into account feedback from the reporter on the appropriateness of our response, but our response will be determined by what will be best for community safety.
+
+Finally, the response team will make a report on the situation to the Systers' Open Source Board. The board may choose to issue a public report of the incident or take additional actions.
+
+## Appealing the response
+
+Only permanent resolutions (such as bans) may be appealed. To appeal a decision of the working group, contact the Systers' Open Source Board at techprogs@anitab.org with your appeal and the board will review the case.
```
Comment (annabauza) [inline .github/REPORTING_GUIDLINES.md:50]: @mayburgos is this a real address techprogs@anitab.org ?

Comment (prachi237) [reply .github/REPORTING_GUIDLINES.md:50]: @annabauza ,  Ma,am  am  I supposed to make any other changes

Comment (annabauza) [reply .github/REPORTING_GUIDLINES.md:50]: Yes this email address no longer exists. Please replace with opensource@anitab.org

**Review by annabauza — CHANGES_REQUESTED** [review]
Please replace deprecated email address with actual one.

File: .github/REPORTING_GUIDLINES.md (hunk 1 above)
Comment (prachi237) [reply .github/REPORTING_GUIDLINES.md]: I changed the email address . Done

**Review by annabauza — APPROVED** [review]

**Pull request merged**